sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -570,7 +578,9 @@ static std::vector<RT::PiProgram> getDeviceLibPrograms(
   // particular program in order to allow us do a more fine-grained check here.
   // Require *all* possible devicelib extensions for now.
   std::pair<DeviceLibExt, bool> RequiredDeviceLibExt[] = {
-      {cl_intel_devicelib_assert, /* is fallback loaded? */ false}
+      {cl_intel_devicelib_assert, /* is fallback loaded? */ false},
+      {cl_intel_devicelib_math, /* is fallback loaded? */ false},
```

Comment:
I think you can omit this "is fallback loaded?" comment.

Reply:
Fixed.